src/gocert.py

```diff
@@ -113,3 +131,70 @@ def create_first_user(self, username: str, password: str) -> int | None:
         logger.info("created the first user in GoCert.")
         id = req.json().get("id")
         return int(id) if id else None
+
+    def get_certificate_requests_table(self, token: str) -> CertificateRequests | None:
+        """Get all certificate requests table from GoCert."""
+        try:
+            res = requests.get(
+                f"{self.url}/api/{self.API_VERSION}/certificate_requests",
+                verify=self.ca_path,
+                headers={"Authorization": f"Bearer {token}"},
+            )
+            res.raise_for_status()
+        except requests.RequestException as e:
+            logger.warning(
```

Comment:
I might be wrong here but should we use `error` instead?

Reply:
I think both are valid, but it makes sense to just do error. changed.